src/interface.h

```diff
@@ -91,6 +91,14 @@ bool em_consp(emacs_env *env, emacs_value cell);
  */
 emacs_value em_car(emacs_env *env, emacs_value cell);
 
+/**
+ * Call (car cell) in Emacs.
+ * @param env The active Emacs environment.
+ * @param cell the cell to get the car of.
+ * @return the car of the cell or nil.
+ */
+emacs_value em_car(emacs_env *env, emacs_value cell);
+
```

Comment:
Looks like duplicated code?

Reply:
Yes, it snuck in at some point and I'm not sure how.  I'm seriously considering closing this pull request and reopening it with a clean branch and just the changes I actually need to make.